plextraktsync/trakt/TraktApi.py

```diff
@@ -53,7 +53,7 @@ def me(self):
             return trakt.users.User("me")
         except (OAuthException, ForbiddenException) as e:
             logger.fatal(f"Trakt authentication error: {str(e)}")
-            raise ClickException(e)
+            sys.exit(1)
```

Comment:
sys.exit in a class? that's a no!

class methods should throw, and this is what it is already doing now.

Reply:
Ok but at the moment we have 3 cascaded exceptions throwed if bad trakt authentication :
OAuthException > ClickException > TypeError

Isn't that bad practice ?

My first fix to remove TypeError was :
```diff
- raise ClickException(e)
+ raise ClickException(str(e))
```

Reply:
you should add also to cli.py:

```py
            except ClickException as e:
                raise ClickException(f"Error running {name} command: {e}")
```

before:
```py
            except Exception as e:
                from plextraktsync.factory import logger
                logger.exception(e)

```

so, that `ClickException` is exiting without printing the trace (too noisy).